### PR TITLE
Add Docker commands for Apple Silicon

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

File: pages/docker_compose.md [hunk 1]
```diff
@@ -77,13 +77,16 @@ __Solution 2__
 
 To build a Docker image of your application using [Jib](https://github.com/GoogleContainerTools/jib) connecting to the local Docker daemon:
 
-- With Maven, type: `./mvnw package -Pprod verify jib:dockerBuild`
-- With Gradle, type: `./gradlew -Pprod bootJar jibDockerBuild`
+- NPM: `npm run java:docker`
+  - On Apple Silicon: `npm run java:docker:arm64`
```
Comment (mraible) [inline pages/docker_compose.md:81]:
```suggestion
- NPM: `npm run java:docker`, on Apple Silicon: `npm run java:docker:arm64`
```

File: pages/docker_compose.md [hunk 2]
```diff
@@ -77,13 +77,15 @@ __Solution 2__
...
-- With Gradle, type: `./gradlew -Pprod bootJar jibDockerBuild`
+- NPM: `npm run java:docker`, on Apple Silicon: `npm run java:docker:arm64`
+- Maven: `./mvnw package -Pprod verify jib:dockerBuild`
+- Gradle: `./gradlew -Pprod bootJar jibDockerBuild`
 
 To build a Docker image of your application without Docker and push it directly into your Docker registry, run:
 
-- With Maven, type: `./mvnw package -Pprod verify jib:build`
-- With Gradle, type: `./gradlew -Pprod bootJar jib`
+- Maven: `./mvnw package -Pprod verify jib:build`
```
Comment (mraible) [inline pages/docker_compose.md:86]:
```suggestion
- Maven: `./mvnw package -Pprod verify jib:build -Dimage=<dockerhub-username>/<artifact-id>`
```

File: pages/docker_compose.md [hunk 3]
```diff
@@ -77,13 +77,15 @@ __Solution 2__
...
+- Gradle: `./gradlew -Pprod bootJar jib`
```
Comment (mraible) [inline pages/docker_compose.md:87]:
```suggestion
- Gradle: `./gradlew -Pprod bootJar jib --image=<dockerhub-username>/<artifact-id>`
```

File: pages/docker_compose.md [hunk 4]
```diff
@@ -77,13 +77,15 @@ __Solution 2__
...
+- On Apple Silicon, change `-Pprod` to `-Pprod,jibArchitecture=arm64`
```
Comment (mraible) [inline pages/docker_compose.md:88]:
```suggestion
```

File: pages/docker_compose.md [hunk 5]
```diff
@@ -77,13 +77,14 @@ __Solution 2__
...
+- NPM: `npm run java:docker`, on Apple Silicon: `npm run java:docker:arm64`
+- Maven: `./mvnw package -Pprod verify jib:dockerBuild`
+- Gradle: `./gradlew -Pprod bootJar jibDockerBuild`
 
 To build a Docker image of your application without Docker and push it directly into your Docker registry, run:
 
-- With Maven, type: `./mvnw package -Pprod verify jib:build`
-- With Gradle, type: `./gradlew -Pprod bootJar jib`
+- Maven: `./mvnw package -Pprod verify jib:build -Dimage=<dockerhub-username>/<artifact-id>`
+- Gradle: `./gradlew -Pprod bootJar jib --image=<dockerhub-username>/<artifact-id>`
```
Comment (mraible) [inline pages/docker_compose.md:87]:
```suggestion
- Gradle: `./gradlew -Pprod bootJar jib -Djib.to.image=<dockerhub-username>/<artifact-id>`
```

File: pages/docker_compose.md [hunk 6]
```diff
@@ -77,13 +77,14 @@ __Solution 2__
...
-- With Gradle, type: `./gradlew -Pprod bootJar jibDockerBuild`
+- NPM: `npm run java:docker`, on Apple Silicon: `npm run java:docker:arm64`
+- Maven: `./mvnw package -Pprod verify jib:dockerBuild`
+- Gradle: `./gradlew -Pprod bootJar jibDockerBuild`
 
 To build a Docker image of your application without Docker and push it directly into your Docker registry, run:
 
-- With Maven, type: `./mvnw package -Pprod verify jib:build`
-- With Gradle, type: `./gradlew -Pprod bootJar jib`
+- Maven: `./mvnw package -Pprod verify jib:build -Dimage=<dockerhub-username>/<artifact-id>`
```
Comment (mraible) [inline pages/docker_compose.md:86]:
```suggestion
- Maven: `./mvnw package -Pprod verify jib:build -Djib.to.image=<dockerhub-username>/<artifact-id>`
```

**Review by mshima — APPROVED** [review]
LGTM.

To publish both amd64 and arm64 images, need to add both platforms:
https://github.com/jhipster/jhipster-registry/blob/d4de3449db4f8308a1f527e10138ee708957e5de/pom.xml#L539-L548

**Pull request merged**